xtask/src/release.rs

```diff
@@ -78,8 +158,18 @@ impl flags::Release {
         Ok(changes.trim().to_owned())
     }
 
+    /// Get the name of this release.
+    fn get_name(&self) -> String {
+        format!("{} {}", self.name, self.version)
+    }
+
+    /// Get the path of the crate for this release.
+    fn get_path(&self) -> PathBuf {
```

Comment:
```suggestion
    fn crate_root(&self) -> PathBuf {
```

getters don't usually have a name starting in `get_` in Rust.